src/utils/apiRequests/api.ts

```diff
@@ -307,6 +291,31 @@ export async function putAuthenticatedRequest<T>(
     console.error('Error 401: You are not Authorized!');
   }
 }
+export async function postRequest(
+  url: any,
+  data: any,
+  errorHandler?: Function,
+  redirect?: string
+): Promise<any> {
+  const res = await fetch(process.env.API_ENDPOINT + url, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-Type': 'application/json',
+      'tenant-key': `${TENANT_ID}`,
+      'X-SESSION-ID': await getsessionId(),
+      'x-locale': `${
+        localStorage.getItem('language')
+          ? localStorage.getItem('language')
+          : 'en'
+      }`,
+    },
+  });
+  const result = await res.json();
+  handleApiError(res.status, result, errorHandler, redirect);
+  return result;
+}
+
```

Comment:
Any reason `postRequest()` has been moved here? There aren't any code changes.

Reply:
Not sure about your intention of reordering the methods, but if you tried to separate authenticated from unauthenticated methods, you did not completely succeed :-)

Before the methods where sorting for the request method like "GET", "POST", "PUT" et al
<img width="492" alt="before" src="https://user-images.githubusercontent.com/1532418/222895500-22d27c5e-316d-4b5c-881b-0b116199516b.png">

Afterwards the authenticated methods are grouped together, but splitting the other methods above and below:
<img width="503" alt="after" src="https://user-images.githubusercontent.com/1532418/222895414-63c50b25-3c9e-43b3-a5e0-f2eb550b3f5c.png">